GithubRepo/Scenes/Search/SearchViewController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  SearchViewController.swift
+//  GithubRepo
+//
+//  Created by Marcos Alves on 07/09/21.
+//
+
+import UIKit
+
+class SearchViewController: UIViewController {
+    
+    var searchTimer: Timer?
+    
+    private let kRepositoryTableViewCellIdentifier = String(describing: RepositoryTableViewCell.self)
+    
+    var searchRepositories: SearchRepoResponse = SearchRepoResponse(totalCount: 0, incompleteResults: false, items: [])
```

Comment:
you don't need to set `SearchRepoResponse` type explicitly as you are already instantiating